neo4j-cypher-dsl/src/main/java/org/neo4j/cypherdsl/core/DefaultStatementBuilder.java

```diff
@@ -60,7 +60,7 @@ class DefaultStatementBuilder implements StatementBuilder,
 	/**
 	 * Current list of reading or update clauses to be generated.
 	 */
-	private final List<Visitable> currentSinglePartElements = new ArrayList<>();
+	final List<Visitable> currentSinglePartElements = new ArrayList<>();
```

Comment:
could be still
```suggestion
	private final List<Visitable> currentSinglePartElements = new ArrayList<>();
```
or am I missing something?